kotlin-native/litmus/src/commonMain/kotlin/CustomTests.kt

```diff
@@ -0,0 +1,95 @@
+import kotlinx.coroutines.sync.Mutex
+import kotlinx.coroutines.sync.withLock
+
+class KindaMP : BasicLitmusTest("kinda mp") {
+
+    var x = 0
+    var y = 0
+
+    override suspend fun actor1() {
+        y = 5
```

Comment:
why `5` ?)
It would make sense to stick to some simple strategy/styleguide when writing litmus tests. 
E.g. use same set of values (i.e. `0`, `1`, etc) consistently.